static/app/components/events/eventTags/eventTagsTree.spec.tsx

```diff
@@ -67,24 +77,112 @@ describe('EventTagsTree', function () {
+
+    const columns = screen.queryAllByTestId('tag-tree-column');
+    expect(columns).toHaveLength(COLUMN_COUNT);
+
+    const linkDropdowns = screen.queryAllByLabelText('Tag Actions Menu');
+    expect(linkDropdowns).toHaveLength(tags.length);
+
+    for (const link of linkDropdowns) {
+      await userEvent.click(link);
+      expect(
+        screen.getByLabelText('View issues with this tag value')
+      ).toBeInTheDocument();
+      expect(
+        screen.getByLabelText('View other events with this tag value')
+      ).toBeInTheDocument();
+    }
   }
 
-  it('renders tag tree with query param', function () {
+  it('renders tag tree with query param', async function () {
     router.location.query.tagsTree = '1';
     render(<EventTags projectSlug={project.slug} event={event} />, {
       organization,
       router,
     });
-
-    assertNewTagsView();
+    await assertNewTagsView();
   });
 
-  it("renders tag tree with the 'event-tags-tree-ui' feature", function () {
+  it("renders tag tree with the 'event-tags-tree-ui' feature", async function () {
     const featuredOrganization = OrganizationFixture({features: ['event-tags-tree-ui']});
     render(<EventTags projectSlug={project.slug} event={event} />, {
       organization: featuredOrganization,
     });
+    await assertNewTagsView();
+  });
+
+  it('renders release tag differently', async function () {
+    const releaseVersion = 'v1.0';
+    const featuredOrganization = OrganizationFixture({features: ['event-tags-tree-ui']});
 
-    assertNewTagsView();
+    const reposRequest = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/repos/`,
+      body: [],
+    });
+    const releasesRequest = MockApiClient.addMockResponse({
+      url: `/projects/${organization.slug}/${project.slug}/releases/${releaseVersion}/`,
+      body: [],
+    });
+    const deploysRequest = MockApiClient.addMockResponse({
+      url: `/organizations/${organization.slug}/releases/${releaseVersion}/deploys/`,
+      body: [],
+    });
+
+    const releaseEvent = EventFixture({
+      tags: [{key: 'release', value: releaseVersion}],
+    });
+    render(<EventTags projectSlug={project.slug} event={releaseEvent} />, {
+      organization: featuredOrganization,
+    });
+    const versionText = screen.getByText<
+      HTMLElement & {parentElement: HTMLAnchorElement}
+    >(releaseVersion);
+    const anchorLink = versionText.parentElement;
+    expect(anchorLink.href).toContain(
+      `/organizations/${organization.slug}/releases/${releaseVersion}/`
+    );
+    expect(reposRequest).toHaveBeenCalled();
+    expect(releasesRequest).toHaveBeenCalled();
+    expect(deploysRequest).toHaveBeenCalled();
+    const dropdown = screen.getByLabelText('Tag Actions Menu');
+    await userEvent.click(dropdown);
+    expect(screen.getByLabelText('View this release')).toBeInTheDocument();
+  });
+
+  it('renders unique links for specific tags', async function () {
```

Comment:
the unmount() in this test seems like you kinda want an `it.each` or you can wrap the test in the for loop. or leave it how it is https://jestjs.io/docs/api#testeachtablename-fn-timeout